cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -193,10 +193,17 @@ display.page.search.tab = statistics
 display.page.search.mode = fast
 request.ui_dispatch_app = cyences_app_for_splunk
 request.ui_dispatch_view = search
-search = | tstats `cs_summariesonly_endpoint` count min(_time) as firstTime max(_time) as lastTime values(Filesystem.user) as user values(Filesystem.file_path) as file_path from datamodel=Endpoint.Filesystem by Filesystem.dest, Filesystem.file_name \
+search = | tstats `cs_summariesonly_endpoint` count min(_time) as firstTime max(_time) as lastTime from datamodel=Endpoint.Filesystem by Filesystem.dest, Filesystem.file_name, Filesystem.file_path \
 | `drop_dm_object_name(Filesystem)` \
-| rex field=file_name "(?<file_extension>\.[^\.]+)$" | `cs_ransomware_extensions` \
-| eventstats sum(count) as total_files_with_extension by dest, file_extension | where total_files_with_extension>=`cs_ransomware_min_file_extension_count` \
+| rex field=file_path "(?<file_location>.*)[\\\\|\\/]" | rex field=file_name "(?<file_extension>\.[^\.]+)$" \
+| stats sum(count) as count, min(firstTime) as firstTime max(lastTime) as lastTime, values(file_name) as sample_file_names by dest, file_extension, file_location \
+| `cs_ransomware_extensions` \
+| eval sample_file_names=mvindex(sample_file_names,0,4) \
+| sort - count | eval file_location=file_location." (".count.")" \
+| stats sum(count) as total_files_with_extension, min(firstTime) as firstTime max(lastTime) as lastTime, list(file_location) as file_location, values(sample_file_names) as sample_file_names by dest, file_extension \
```

Comment:
I would suggest to remove this stats and just use eventstats to get total number_of_events for particular file_extension same as existing one.

Reply:
In previous stats we have file_location in group by.
Then we are calculating "| sort - count | eval file_location=file_location." (".count.")" "
And then we are just grouping with dest and extension.

Reply:
In previous stats we have file_location in group by.
Then we are calculating "| sort - count | eval file_location=file_location." (".count.")" "
And then we are just grouping with dest and extension.